site/_data/examples.json

```diff
@@ -512,10 +516,6 @@
         "name": "concat_layer_voyager_result",
         "title": "Comparative Likert Scale Ratings",
         "description": "Comparing Likert scale ratings between two conditions. (Figure 10. from @kanitw et al.'s \"Voyager 2: Augmenting Visual Analysis with Partial View Specifications\" -- http://idl.cs.washington.edu/files/2017-Voyager2-CHI.pdf)."
-      },
-      {
```

Comment:
We had this example already in the gallery.

Reply:
It's ok to show an example twice, as long as they appear in the right sections.

Reply:
It was twice in the same section. 

<img width="1077" alt="Screen Shot 2020-03-24 at 15 28 19" src="https://user-images.githubusercontent.com/589034/77482909-2d684280-6de4-11ea-8ddf-e7b1523c069e.png">

Reply:
hahaha ok.